### ISSUE TITLE
Have the ability to add a canonical link tag, on any selected page, pointing to themselves like in the example above, or to another internal page URL or even an external URL

### ISSUE THREAD

**Issue by GormFrank** [wet-boew/wet-boew#8877]
Canonical is useful for crawlers and good for SEO in general.

The canonical link tag (referred in the text as rel canonical) is a way of telling robots and search engines that a specific URL represents the original or master copy of a page. The use of the rel canonical prevents problems caused by duplicate content appearing on multiple URL.

Even on one page, you agree with me that all of those are the same page:

- https://www.canada.ca/en.html
- http://www.canada.ca/en.html
- https://canada.ca/en.html
- https://canada.ca/en/
- https://canada.ca/en.html?utm_campaign...

Well for crawlers they are all considered as different pages, and canonical sets one as the main one.

Also, rel canonical can point to URLs on other websites. For instance, sometimes news sites post articles that are duplicates or very similar to other ones on other websites, and so they can refer the original article there through the rel canonical.

It is good practice to have every single page with a rel canonical, even if it is pointing to itself. So this should be the default on every single page on canada.ca; to have the rel canonical point to itself. 

For example, the following would be the rel canonical for the english home page:
<link rel="canonical" href="https://www.canada.ca/en.html" />

Language alternatives are not considered as duplication so the same page in French can have the rel canonical on itself too. Again, for the French home page we would have:
<link rel="canonical" href="https://www.canada.ca/fr.html" />

**NOTE:** However, there is a way to tag languages alternatives, as explained in issue #8878...

To recap, the minimum would be to have the ability to add a rel canonical, on any selected page, pointing to themselves like in the example above, or to another internal page URL or even an external URL. 

Resources:
https://yoast.com/academy/technical-seo-training/
https://moz.com/learn/seo/canonicalization
